### ISSUE TITLE
Remove duplicate views between Calypso and WordPress Admin

### ISSUE THREAD

**Issue by richtabor** [Automattic/wp-calypso#95478]
### Brief 

This project aims to eliminate duplicate views between the Calypso interface and the WordPress admin (WordPress admin) interface to streamline the user experience and reduce confusion. Currently, users encounter similar or overlapping functionalities in both Calypso and WordPress admin, which lead to inconsistencies in managing their sites.

### Problem 

Having two separate interfaces (Calypso and wp-admin) with similar features creates redundancy and confusion for users. This duplication complicates workflows, requiring users to navigate and manage their sites across two systems. It also increases maintenance efforts, as both interfaces need to be updated and maintained separately.

### Proposal

- Determine if this should only function for new accounts. 
- Identify views that are duplicated between Calypso and WordPress admin.
- Remove redundant views from one Calypso and consolidate the features into WordPress admin where they align with the core WordPress experience.
- Ensure any unique features in Calypso that are not in WordPress admin are retained only where necessary after defining any necessary requirements. 
- Identify any needs to improve in Core in order to switch.

**Comment by tanjoymor** [comment]
The following notes apply to all of the sub-issues. Additional notes are added per sub-issue.

## Calypso views
**Recommendation:** Provide an in-product notice on every Calypso screen that will be switching to the Core WP Admin view, in advance of shipping the changes for all All users / All platforms. Will need further discussion and exploration regarding design and copy.

**Suggested copy:** This screen will switch to the Classic view on DATE. Click the View tab (top right) to switch to Classic now. [Learn more](https://wordpress.com/support/navigate-wordpress-com/). Switch to the full Classic Admin Interface under Settings > General. [Learn more](https://wordpress.com/support/set-your-admin-interface-style/).

[Side note that I've also made suggestions regarding the docs linked above in the [docs GH Issue 2693](https://github.com/Automattic/en.support-docs-content/issues/2963).]

**Reasons:** 
* Some existing Dotcom users are not familiar with WP Admin at all, some have never even seen these screens. 
* Some existing Dotcom users _prefer_ the Calypso views over the WP Admin views. This gives them a heads up and time to adjust.
* New users (while we work on this) might be confused by Calypso views and it will draw their attention to the Classic WP Admin view.
* It will give new users confidence that we're transparent and communicative about the changes we make in our platform.

### After shipping
**Recommendation:** Provide a notice, in the form of a tooltip, banner, or pop-up modal, about the changes, for a defined period of time. This could be on-login or per screen. Will need further discussion and exploration regarding design and copy.

**Reasons:**

* It will reduce confusion and questions from existing users who missed the advance notices.
* It will give new users confidence that we're transparent and communicative about the changes we make in our platform.

## WP Admin views
**Observation:** “Help” tab (top right) pulls directly from Dotorg and only includes Dotorg resources. 

**Recommendation:** Hide this tab for All users / All platforms

**Reasons:** 
* Dotorg links will cause confusion for existing Dotcom users.
* These Help tab links open in the same screen, taking users away from Dotcom and their site.
* Having both the Dotorg “Help” tab and the ? icon for the Dotcom Help Center is confusing.

**Questions for possible exploration/consideration:** 

Is it possible for Dotcom to override the information in the Help tab with Dotcom resources? 
* Do we need to?
-- Can the presence of Dotorg resources be better explained?
* Should we be removing Dotcom Support docs for docs that exist in Dotorg?
-- Are Dotorg resources kept updated in a timely manner?
-- Also noting that Dotcom Support docs do offer upsell opportunities that we don't have in Dotorg docs, even though there is some duplication of information.
-- Should Dotcom Support docs be cross-linking to their Dotorg counterparts?
* Should we include a combination of both Dotcom and Dotorg resources, and the forums?

_These questions teeter on being out of scope for this project, but wanted to raise them for awareness and consideration. They are relevant to in-product work and documentation._

**Observation:** In WP Admin the folder types are hidden when empty on screens like Pages and Posts. This has potential to be confusing for Dotcom users. The WP Admin Comments screen shows the empty folder types. Maybe a suggestion for Core to make this behavior consistent?


Related resources:
[Figma](https://www.figma.com/board/quAcTGOFbZBAuE5h6INijK/Dotcom-Screens-Audit?node-id=24-18&node-type=canvas&t=9ax66jIEMI9kEcPZ-0) 
[Google Doc notes](https://docs.google.com/document/d/1uSEK-wcOm4SwcNwpyOQRc8h_3tLeb-2XLq3ewfbq2jo/edit?tab=t.nt3ydfho2e3j)

**Comment by Copons** [comment]
@richtabor @annezazu I've replied to all current issues with recommendations for moving forward, and I've added another issue for "Tools > Export".

AFAICS the only missing duplicate screen is Settings, and it's a big one.
I've started an audit comparing Simple Calypso, Simple wp-admin, and Atomic wp-admin. However, I suspect someone already did it. Maybe @taipeicoder knows more? This seems up Lego's alley. 🙂

**Comment by Copons** [comment]
I've also summarized everything on P2: pfYzsZ-E4-p2

**Comment by taipeicoder** [comment]
> I've started an audit comparing Simple Calypso, Simple wp-admin, and Atomic wp-admin. However, I suspect someone already did it.

Left a reply in the P2 post pfYzsZ-E4-p2#comment-536

**Comment by jameskoster** [comment]
I acknowledge it's not technically a 'duplicate' page, but has 'My Home' been discussed anywhere? Should it merge with the wp-admin Dashboard, or does it just need to be refreshed using the wp-admin design language?

**Comment by dsas** [comment]
@jameskoster it was put on pause indefinitely pekYwv-4R6-p2 :

> Deprioritized because of complexity and uncertainty on how it plays with other onboarding initiatives like the Global Launchpad

**Comment by jameskoster** [comment]
Thanks for the link. How does that align with the 'untangling calypso' initiative, it will need to be tackled at some point, no?

**Comment by dsas** [comment]
Absolutely. It's part of that initiative. I think it was just parked because there was plenty of stuff to be getting on with, and it wasn't clear how we were going to do it. 

I think the difficulty was that the future of the launchpad is unclear. The launchpad is currently in onboarding and the home screen and is Calypso based. I'm not sure whether any of that has got any clearer since.

**Comment by ianstewart** [comment]
@Copons @oskosk it's not clear what the status is on the above. Is there an update or any blockers on anything. Some, I think may need "product decisions" (like "are we comfortable swapping out Posts now") but others may need "just work" (like Editor iFrame.) Maybe we could split some issues out into different epics to unblock things and make this issue less complex?

**Comment by Copons** [comment]
@ianstewart we ran a CfT (pdtkmj-3dx-p2) over the holidays which didn't surface any major issue. We are updating the notices design, and we plan to launch the holdout by EOW.

The CfT involved Posts, Categories, Tags, Portfolios, Testimonials, Comments, and even the iframed Editor when accessed through the above post types (I don't consider it a blocker for anything anymore 🙂).

Since then, we also have Pages and My Mailboxes ready-ish; Media and Settings just started and are handled as separate projects.

Of the unassigned/unstarted tasks, only Themes and Plugins are _really_ complex to require their own separate discussion.

**Comment by ianstewart** [comment]
Excellent. Have we started the discussions somewhere for those in a P2?

**Comment by taipeicoder** [comment]
@ianstewart for Calypso Media we have the P2 pfYzsZ-1ci-p2 which aims to align on product decisions based on our research. Next week we will post the PT, the Feedback Request post, and start development.

For Calypso Settings we have the research doc 1FK_BUYtN745SBftXuXFXJdwnkZPw71GJbXs_v2YtSGA-gdoc. Next week we will the PT, the Feedback Request post, and start development.

**Comment by ianstewart** [comment]
@Copons I'd like to set an ETA for this issue. It may be monumental and that might prompt us to chunk up and split the work out — but I'd like a date to track against.

**Comment by Copons** [comment]
> I'd like to set an ETA for this issue. It may be monumental and that might prompt us to chunk up and split the work out — but I'd like a date to track against.

@ianstewart I'm treating this as a tracking issue rather than a project.

There are a few "smaller" tasks that like Tools -> Import and Export, and then currently 7 epics that are standalone projects, some not yet started.

As you can imagine, the estimation would be very different if we count only the remaining smaller tasks, or all the projects. Do you have a preference?

**Comment by tbradsha** [comment]
As Calypso pages are removed, can whoever is doing it make sure the appropriate tests are also removed? Media pages caused a bunch of failures until those tests were deleted: #99680
